<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMSTemplateDialog</class>
 <widget class="QDialog" name="SMSTemplateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>SMS Template</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fromLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="bodyEdit">
        <property name="placeholderText">
         <string>Preferred length of 160 chars. Messages with more than 160 chars will be split into pieces 153 chars with maximum length of 918 chars.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="plainText">
      <string>You may use the following variables in the subject line or body for the email template:

&lt;LINKS&gt;: hyperlinks for all photos
&lt;ALBUMNAME&gt;: name of album
&lt;DATE&gt;: date of photos taken
&lt;PHONE&gt;: phone number of guest</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SMSTemplateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SMSTemplateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
